lib/Interpreter/CppInterOp.cpp

```diff
@@ -2629,6 +2629,7 @@ namespace Cpp {
 #ifdef _WIN32
     // FIXME : Workaround Sema::PushDeclContext assert on windows
     ClingArgv.push_back("-fno-delayed-template-parsing");
+    ClingArgv.push_back("-femulated-tls");
```

Comment:
Clang-repl should support proper tls.

Reply:
Will remove. This was added the output suggested it needs access to emulated tis when trying to write to the standard output.